.github/workflows/cicd.yml

```diff
@@ -0,0 +1,32 @@
+name: cicd
+
+on:
+  push:
+    branches:
+      - v3.0.0-openmldb
+  pull_request:
+
+jobs:
+  build_spark_distribution:
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/4paradigm/hybridsql:0.4.1
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-java@v2
+        with:
+          distribution: 'adopt'
+          java-version: '8'
+
+      - name: Cache local Maven repository
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('java/**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+      - name: compile
+        run: |
+          ./dev/make-distribution.sh --name openmldbspark --pip --tgz -Phadoop-2.7 -Pyarn
```

Comment:
missing release step

Reply:
We will add release later, maybe in another job.